localstack/testing/snapshots/transformer_utility.py

```diff
@@ -165,18 +165,36 @@ def sqs_api():
         """
         return [
             TransformerUtility.key_value("ReceiptHandle"),
-            TransformerUtility.key_value("SenderId"),
+            # account ID
+            TransformerUtility.key_value(
+                "SenderId", value_replacement="111111111111", reference_replacement=False
+            ),
             TransformerUtility.jsonpath("$..MessageAttributes.RequestID.StringValue", "request-id"),
             KeyValueBasedTransformer(_resource_name_transformer, "resource"),
-            KeyValueBasedTransformer(_signing_cert_url_token_transformer, replacement="token"),
+        ]
+
+    @staticmethod
+    def sns_api():
+        """
+        :return: array with Transformers, for sns api.
+        """
+        return [
+            TransformerUtility.key_value("SequenceNumber"),  # this might need to be in SQS
+            TransformerUtility.key_value(
+                "Signature", value_replacement="<signature>", reference_replacement=False
+            ),
```

Comment:
should we maybe just use the default `value_replacement`?

Reply:
with the reference_replacement set to false, the default value would be `signature`. I think it's fine to use  `<signature>` here as it indicates at the first sight that the value was replaced.

Reply:
Yes that was my issue, the fact that the `<[...]>` were missing was troubling me, so I added it manually!